portal/contacts/models.py

```diff
@@ -122,19 +139,22 @@ def __str__(self):
 class CommunicationRecord(models.Model):
 
     internal_contact = models.ForeignKey(
-        Contact,
+        Individual,
         related_name="internal_contact"
     )
-    external_contact = models.ForeignKey(Contact)
+
+    # TODO: Make at least one required?
```

Comment:
@RachellCalhoun Do you know?

Reply:
We can test this with: https://github.com/arts-in-motion/arts-in-motion/issues/48